WordPress-iOS-Shared/Core/WPStyleGuide.m

```diff
@@ -139,111 +140,120 @@ + (UIFont *)tableviewSectionHeaderFont
     return [WPFontManager openSansBoldFontOfSize:12.0];
 }
 
+
 #pragma mark - Colors
+// https://wordpress.com/design-handbook/colors/
 
-+ (UIColor *)wordPressBlue
++ (UIColor *)colorWithR:(NSInteger)red G:(NSInteger)green B:(NSInteger)blue alpha:(CGFloat)alpha
```

Comment:
I should add, when i see the `colorWithR:` I still pause for a second to think and translate R in my head to Red whereas when I see `colorWithRed:` or `colorWithRedValue:` i don't really stop to think about it as it reads more naturally and doesn't need any translation.